tests/boards/tenstorrent/tt_blackhole/sensor/pvt/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_ZTEST=y
+
+CONFIG_SENSOR=y
+CONFIG_RTIO=y
+CONFIG_SENSOR_ASYNC_API=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
```

Comment:
This seems a little odd - why would we need Newlib specifically?

Reply:
That was for float-printing in the tests, but I'm still playing around with the code so I might not need that